### ISSUE TITLE
Limit only works on picking images, but not in select.

### ISSUE THREAD

**Issue by tylik1** [rvera/image-picker#92]
When limit is set, you still can pick more images from multiselect, which is probably not right.

**Comment by rvera** [comment]
Good point, I always pictured the select element to be hidden (since the idea is that this replaces that). Will try to also implement this.
